### DISCUSSION TITLE
SUBMIT a notice

### DISCUSSION THREAD

**Discussion by ismailatitallah** [OP-TED/eForms-SDK#217] [Q&A] [answered]
Hello,

I am trying to submit a notice with https://enotices2.preview.ted.europa.eu/ api. I created my api-key from the web-site https://developer.preview.ted.europa.eu/ted-api . And I added this api-key to my api call in the header : "Bearer xxxxxxx". But I am getting **403 Forbidden** as response.

I don't know why i am having 403. Can any one help me?

**Comment by pcolast** [comment, marked as answer]
Hello,

Before starting using the API, it is needed to log into eNotices2, so that eNotices2 account is created. 
The same EU login needs to be used as used to generated the TEDAPI key.

Regards,
Stefan
